### PR TITLE
Add PageLanguage and PerPageLanguage to MW extensions

### PR DESCRIPTION
**Pull request by brandon-wm**

### CODE REVIEW

**Comment by clarfonthey** [comment]
Definitely looks fine for PageLanguage (WMF supported) but my only concern with PerPageLanguage is that we prefer updating to the latest MW version when it comes out, and they might lag behind in support.

File: ManageWikiExtensions.php [hunk 1]
```diff
@@ -2589,9 +2589,22 @@
...
+		'linkPage' => 'https://www.mediawiki.org/wiki/Special:MyLanguage/Extension:PageLanguage',
+		'conflicts' => false,
+		'requires' => [],
+		'section' => 'other',
+	],
+	'perpagelanguage' => [
+		'name' => 'PerPageLanguage',
+		'linkPage' => 'https://www.mediawiki.org/wiki/Special:MyLanguage/Extension:PerPageLanguage',
+		'conflicts' => false,
+		'requires' => [],
```
Comment (Universal-Omega) [inline ManageWikiExtensions.php:2603]:
```suggestion
		'requires' => [],
		'install' => [
			'settings' => [
				'wgPageLanguageUseDB' => true,
			],
		],
```

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: ManageWikiExtensions.php [hunk 2]
```diff
@@ -2589,9 +2589,27 @@
 		'requires' => [],
 		'section' => 'other',
 	],
+	'pagelanguage' => [
+		'name' => 'PageLanguage',
```
Comment (Universal-Omega) [inline ManageWikiExtensions.php:2593]:
```suggestion
		'name' => 'Page Language',
```

**Review by Universal-Omega — CHANGES_REQUESTED** [review]
'name' must match the name in extension.json

**Review by Universal-Omega — APPROVED** [review]

**Pull request merged**